portal_odoo_debranding/static/description/index.html

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This line should be removed otherwise you get a warning error when installing the module : 
`WARNING odoo17 py.warnings: /home/user/dev/odoo17/ocb/odoo/addons/base/models/ir_module.py:178: DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, portal_odoo_debranding can just have a UTF8 description with not need for a declaration.`

Note : this was fixed in maintainer-tools with https://github.com/OCA/maintainer-tools/pull/596 back in December 23 so it should not be present anymore when pre-commit reissues the index.html

Reply:
```suggestion

```

Reply:
thx. I updated maintainertools and pre-commit, resulting in a conflict in pre-commit-config.yaml.